ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -488,7 +498,21 @@ class WalletHomeScreen extends React.PureComponent<Props> {
         contextualHelpMarkdown={contextualHelpMarkdown}
         faqCategories={["wallet", "wallet_methods"]}
       >
-        {this.newMethodAdded ? this.newMethodAddedContent : transactionContent}
+        {this.newMethodAdded ? (
+          this.newMethodAddedContent
+        ) : (
+          <React.Fragment>
+            {/* Item displays only if the flag is enabled */}
+            {this.props.isBonusEnabled && (
```

Comment:
This flag already exists https://github.com/pagopa/io-app/blob/9b2a7e59d2b0a14c6c875652a1e0c05465371812/ts/config.ts#L45